### ISSUE TITLE
Typescript Conversion

### ISSUE THREAD

**Issue by NullVoxPopuli** [exivity/react-orbitjs#18]
This library has some pretty involved cache checking/busting mechanisms. It'd be great to have a type-system have contributors' backs as they make changes. :)

I can do this and submit a PR, but imo: https://github.com/exivity/react-orbitjs/pull/16 should be merged first :)

**Comment by hongaar** [comment]
That'd be very welcome. I've just reviewed your PR in more detail so we can merge asap.

**Comment by hongaar** [comment]
Closing in favour of #25.
